google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentRunConfigurationEditor.java

```diff
@@ -386,10 +387,18 @@ private void validateConfiguration() throws ConfigurationException {
         throw new ConfigurationException(
             GctBundle.message("appengine.flex.config.custom.dockerfile.error"));
       }
-    } else if (environment.isStandard() && CloudSdkService.getInstance().validateCloudSdk()
-        .contains(CloudSdkValidationResult.NO_APP_ENGINE_COMPONENT)) {
-      throw new ConfigurationException(
-          CloudSdkValidationResult.NO_APP_ENGINE_COMPONENT.getMessage());
+    } else {
+      Set<CloudSdkValidationResult> validationResults =
+          CloudSdkService.getInstance().validateCloudSdk();
+      if (validationResults.contains(CloudSdkValidationResult.CLOUD_SDK_NOT_FOUND)) {
+        throw new ConfigurationException(
+            GctBundle.message("appengine.cloudsdk.location.invalid.message.deploy"));
+      }
+      if (environment.isStandard()
```

Comment:
I think this can be refactored a bit for clarity. e.g. move the standard check up one level.